package/gluon-config-mode-autoupdater-freifunk/Makefile

```diff
@@ -0,0 +1,14 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-config-mode-autoupdater-freifunk
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/gluon-config-mode-autoupdater-freifunk
```

Comment:
gluon-config-mode-autoupdater-consent

This package should not be Freifunk-specific only